modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -118,8 +118,102 @@ definition {
 		}
 	}
 
-	@description = "LPS-178192. Verifies Expanded Text effects toggle can be enabled."
-	@priority = 4
+	@description = "LPS-178192."
+	@priority = 5
+	test CanBeDisabledByInstanceSettings {
+		property portal.acceptance = "true";
+
+		task ("Given user with admin privileges") {
+			Navigator.openURL();
+		}
+
+		task ("And “Enable Accessibility Menu” is enabled") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Configuration",
+				panel = "Control Panel",
+				portlet = "Instance Settings");
+
+			Click(
+				key_itemName = "Accessibility",
+				locator1 = "ListGroupItem#ITEM_TEXT");
+
+			AssertChecked.assertCheckedNotVisible(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+		}
+
+		task ("When turn off Enable Accessibility Menu") {
+			Uncheck.uncheckToggleSwitch(
+				checkboxName = "Enable Accessibility Menu",
+				locator1 = "Checkbox#ANY_CHECKBOX");
+
+			Click(locator1 = "Button#UPDATE");
+
+			Alert.viewSuccessMessage();
+
+			Refresh();
+		}
+
+		task ("Then Accessibility Menu cannot be accessed") {
+			AccessibilityMenu.goToAccessibilityMenuQuickLink();
+
+			AssertElementNotPresent(
+				key_link = "Open Accessibility Menu",
+				locator1 = "AccessibilityMenu#QUICK_LINK");
+		}
+	}
+
+	@description = "LPS-178192. Verifies it can be enabled by Site Settings."
+	@priority = 3
+	test CanBeEnabledBySiteSettings {
+		property portal.acceptance = "false";
+
+		task ("Given Accessibility Menu is disabled") {
+			Navigator.openURL();
+
+			AccessibilityMenu.deactivateAccessibilityMenu(portlet = "Instance Settings");
+		}
```

Comment:
Why is this step necessary? I remember Evan saying that there's a reset default properties option instead of disabling. Why don't we use that?

Reply:
I  haven't found it, so I did this workaround which I already used at other tests... I'll take a closer look at this part as soon as I start work tomorrow morning